helpdesk/akismet.py

```diff
@@ -31,10 +31,10 @@
 Whatever you pass in, will replace the *Python Interface by Fuzzyman* part.
 **0.2.0** will change with the version of this interface.
 
-Usage example::
```

Comment:
The second colon is needed for rst literal-block syntax and should be restored.

While we're at it, line 27 above should also end with just the double colons, the single colon and surrounding spaces that precede them should be removed so the line ends with just "is::".